test/src/test/java/org/corfudb/runtime/clients/NettyCommTestUtil.java

```diff
@@ -111,8 +149,7 @@ public static CertificateManager build(
             );
 
             return new CertificateManager(
-                    gen, certName, chain, cert, keyStore, password, alias, jksPath, keyStoreConfig,
-                    trustStoreManager, cfg
+                    chain, newCert, keyStore, password, alias, jksPath, keyStoreConfig, trustStoreManager, cfg
```

Comment:
I know it was already there, but since args are more, maybe convert this into a `@Builder`?

Reply:
The build method contains a lot of custom logic, so can't make it builder. 
Actually I can, but it will be more awkward solution, so AllArgsConstructor works for us